### PR TITLE
feat(ui5-popover): implement hide-block-layer property

### LINKED ISSUE #2406: [Feature Request][Popover] Mask can be turned on or off when modal is enabled
Dear colleagues,
I am from SuccessFactors Web Experience Shanghai team.

Component:
https://sap.github.io/ui5-webcomponents/playground/components/Popover/

Feature request:
Mask can be turned on or off when modal is enabled

### PR DESCRIPTION
**Pull request by fifoosid**
Fixes #2406

### CODE REVIEW

File: packages/main/src/Popover.js [hunk 1]
```diff
@@ -106,6 +106,17 @@ const metadata = {
 			type: Boolean,
 		},
 
+		/**
+		 * Defines whether the block layer will be shown if modal property is set to true.
+		 * @type {Boolean}
```
Comment (ilhan007) [inline packages/main/src/Popover.js:111]: boolean

**Review by ilhan007 — CHANGES_REQUESTED** [review]
It's worth adding a test, moreover soon  the components will be handed over and it is good if they have decent code coverage

**Review by vladitasev — CHANGES_REQUESTED** [review]
Please rename "block layer" to "backdrop" **only in the public API** as this is what most people would understand. The hide/showBlockLayer functions are already used in third-party popups so don't rename them

**Review by ilhan007 — APPROVED** [review]

**Comment by ilhan007** [comment]
It is fine to m,  @vladitasev  could you check if everything is ok, I removed your review as outdated

**Pull request merged**